views/template-partials/footer.twig

```diff
@@ -26,10 +26,14 @@
               <a href="{{ item.link }}" {{ item.is_target_blank ? 'target="_blank"' }}>{{ item.title }}</a>
             </li>
           {% endfor %}
+          <li>
+              Last update: August 16, 2022
+          </li>
         {% endif %}
       </ul>
       <p>Copyright © The Regents of the University of California, Davis campus. All rights reserved.</p>
     </div>
+    <div class="u-align--center u-space-mt--large">This site is officially grown in <a href="https://sitefarm.ucdavis.edu">SiteFarm</a>.</div>
```

Comment:
this is not true...

Reply:
Kimmy had said everything above the Aggie's logo is ours, but the content below should match the other UC sites. Sorry about that. I've removed that line.